dev-lang/erlang/files/epmd.confd

```diff
@@ -0,0 +1 @@
+command_args="--daemon -relaxed_command_check -address 127.0.0.1"
```

Comment:
Would you mind adding a comment explaining what those default options do above, for the user's convenience?